libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -102,39 +111,41 @@ void HarmonicNotchFilter<T>::init(float sample_freq_hz, float center_freq_hz, fl
     }
 
     _sample_freq_hz = sample_freq_hz;
-    const float nyquist_limit = sample_freq_hz * 0.48f;
+    // Calculate spread required to achieve an equivalent single notch using two notches with Bandwidth/2
+    _notch_spread = bandwidth_hz / (32 * center_freq_hz);
```

Comment:
definitely no chance of a divide by zero?

Reply:
Good catch - I'll move up the constrain_float() below

Reply:
Fixed